pkg/windows/service/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-windows-api-reviewers
+  - mrunalp
+  - kannon92
+  - pacoxu
+  - SergeyKanzhelev
+reviewers:
+  - sig-windows-api-reviewers
+  - sig-node-reviewers
```

Comment:
also add labels:
```suggestion
  - sig-node-reviewers
  
 
labels:
  - sig/windows
  - sig/node
```

Reply:
added

Reply:
thx. I suggest just do sig-node-approvers as suggested above and merge it already. We can add Windows approvers later

Reply:
updated

Reply:
pls remove `sig-windows-api-reviewers` from approvers

Reply:
updated